src/frontend/ExtensionManager/index.tsx

```diff
@@ -26,27 +27,48 @@ const ExtensionManager = function () {
   const rootRef = useRef<HTMLDivElement>(null)
   const trueAsStr = 'false' as unknown as boolean | undefined
 
+  const mmContainerStyle = {} as React.CSSProperties
+
+  if (OverlayState.title === 'HyperPlay Extension') {
```

Comment:
i feel like this is a lot of added complexity for what is still basically a case of string matching. it also only needs to match in two places in the code

I like your idea about the OverlayMode type though. I will add that and change the title type to use that

Reply:
The map is only to leverage typescript in case we add a different mode, we'd be forced to add a title to it (it has happened to me, I forge to add the switch case and the result is an empty string).

Not a blocker.